homeassistant/setup.py

```diff
@@ -134,11 +134,11 @@ def log_error(msg: str, link: Optional[str] = None) -> None:
     # So we do it before validating config to catch these errors.
     try:
         component = integration.get_component()
-    except ImportError:
-        log_error("Unable to import component", integration.documentation)
+    except ImportError as err:
+        log_error("Unable to import component:" + str(err), integration.documentation)
```

Comment:
Please use f-strings, see https://developers.home-assistant.io/docs/en/development_guidelines.html for more information.

Reply:
done